screwdriver.yaml

```diff
@@ -1,5 +1,7 @@
 cache:
-  event: ["~/.m2"]
+  job:
+    pull-commit: ["~/.m2"]
```

Comment:
Not necessary.  job cache is never set by pull request builds.
https://cd.screwdriver.cd/pipelines/6154/builds/683303/steps/sd-teardown-screwdriver-cache-bookend
> 00:00:00 2021/02/27 01:51:19 Skipping set job-scoped cache for Pull Request

However, using pipeline cache would let pull requests read the .m2 saved by the commit job.

Reply:
Let's also add the cache cleanup
```yaml
   - teardown-maven: >
        rm -rf /root/.m2/repository/com/yahoo/fili
```